auth-web/src/views/auth/home/HomeView.vue

```diff
@@ -130,11 +130,23 @@
               <h3 class="mb-6">Contact Us</h3>
               <p class="mb-5">For support or questions about this application, contact us at:</p>
               <ul class="contact-info__list mb-5">
-                <li><span>Toll Free:</span><a :href="`tel:+${$t('techSupportTollFree')}`">{{ $t('techSupportTollFree') }}</a></li>
-                <li><span>Phone:</span> <a :href="`tel:+1${$t('techSupportPhone')}`">{{ $t('techSupportPhone') }}</a></li>
-                <li><span>Email:</span> <a href="mailto:bcregistries@gov.bc.ca?subject=BC Registries - Business Registry Support Request">bcregistries@gov.bc.ca</a></li>
+                <li>
+                  <span>{{ $t('labelTollFree') }}</span>
```

Comment:
Does anyone know why this is wrapped in a "span"? Maybe for accessibility (screen readers)?
I wanted to clean this up, but I didn't want to change too  much or break anything :)